### PR TITLE
Market module

### PR DESCRIPTION
**Pull request by bachstatter**
Adds a Market Module with external view functions needed for UI

### CODE REVIEW

File: markets/perps-market/contracts/modules/PerpsMarketModule.sol [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import "../storage/PerpsMarket.sol";
+import "../storage/MarketConfiguration.sol";
+import "../storage/PerpsPrice.sol";
+import "../storage/AsyncOrder.sol";
+import "../interfaces/IPerpsMarketModule.sol";
+
+contract PerpsMarketModule is IPerpsMarketModule {
+    using PerpsMarket for PerpsMarket.Data;
+
+    function skew(uint128 marketId ) external view override returns (int256){
```
Comment (sunnyvempati) [inline markets/perps-market/contracts/modules/PerpsMarketModule.sol:13]:
```suggestion
    function skew(uint128 marketId) external view override returns (int256){
```

some odd spacing, but weird that lint or prettier isn't catching it.

File: markets/perps-market/contracts/modules/PerpsMarketModule.sol [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        PerpsMarket.Data storage perpsMarket = PerpsMarket.load(marketId);
+        return perpsMarket.size;
+    }
+    function maxOpenInterest(uint128 marketId ) external view override returns (uint256){
+        MarketConfiguration.Data storage marketConfiguration = MarketConfiguration.load(marketId);
+        return marketConfiguration.maxMarketValue;
+    }
+    function currentFundingRate(uint128 marketId ) external view override returns (int){
+        PerpsMarket.Data storage perpsMarket = PerpsMarket.load(marketId);
+        return perpsMarket.currentFundingRate();
```
Comment (sunnyvempati) [inline markets/perps-market/contracts/modules/PerpsMarketModule.sol:27]: can merge into one line

**Review by sunnyvempati — COMMENTED** [review]
run prettier on all these files.  i think i have auto save prettier but the `markets/perps-market` might not be part of the lint/prettier process in CI

File: markets/perps-market/contracts/modules/PerpsMarketModule.sol (hunk 1 above)
Comment (bachstatter) [reply markets/perps-market/contracts/modules/PerpsMarketModule.sol]: Spacing fixed

File: markets/perps-market/contracts/modules/PerpsMarketModule.sol (hunk 2 above)
Comment (bachstatter) [reply markets/perps-market/contracts/modules/PerpsMarketModule.sol]: fixed

**Review by sunnyvempati — APPROVED** [review]

**Pull request merged**